src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -842,7 +842,7 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
         $dumper = new PhpDumper($container);
 
         if (class_exists('ProxyManager\Configuration') && class_exists('Symfony\Bridge\ProxyManager\LazyProxy\PhpDumper\ProxyDumper')) {
-            $dumper->setProxyDumper(new ProxyDumper(substr(hash('sha256', $cache->getPath()), 0, 7)));
```

Comment:
this doesn't serve any purpose now, and made container's cache path-dependend

Reply:
should we deprecate the salt argument in 4.1 ?

Reply:
that's possible, although it doesn't really hurt, so low priority on my side (if anyone wants to do it, PR welcome :))